Tests/LibWasm/test-wasm.cpp

```diff
@@ -62,3 +95,102 @@ TESTJS_GLOBAL_FUNCTION(compare_typed_arrays, compareTypedArrays)
     auto& rhs_array = static_cast<JS::TypedArrayBase&>(*rhs);
     return JS::Value(lhs_array.viewed_array_buffer()->buffer() == rhs_array.viewed_array_buffer()->buffer());
 }
+
+void WebAssemblyModule::initialize(JS::GlobalObject& global_object)
+{
+    Base::initialize(global_object);
+    define_native_function("getExport", get_export);
+    define_native_function("invoke", wasm_invoke);
+}
+
+JS_DEFINE_NATIVE_FUNCTION(WebAssemblyModule::get_export)
+{
+    auto name = vm.argument(0).to_string(global_object);
+    auto this_value = vm.this_value(global_object);
+    auto instance = static_cast<WebAssemblyModule*>(this_value.to_object(global_object));
+    if (!instance) {
+        vm.throw_exception<JS::TypeError>(global_object, "Not a WebAssemblyModule");
+        return {};
+    }
```

Comment:
This doesn't check for `WebAssemblyModule` but any object though? `to_object()` + `is<T>` should probably do the trick